getting-started/installation-docker.md

```diff
@@ -5,7 +5,7 @@
 Start a TimescaleDB instance, pulling our Docker image from [Docker Hub][] if it has not been already installed:
 
 ```bash
-docker run -d --name timescaledb -p 5432:5432 -e POSTGRES_PASSWORD=password timescale/timescaledb:latest-pg:pg_version:
+docker run -d --name timescaledb -p 5432:5432 -e POSTGRES_PASSWORD=password timescale/timescaledb:x.y.z-pg:pg_version:
```

Comment:
Do we want to have as `x.y.z`, or explicitly label it for now as `2.0.0`, and then switch back to `latest` when 2.0 is in GA?  The reason it was latest was so that people could just cut/paste.

Reply:
turns out that `x.y.z` (locally at least) is replaced at runtime with the branch of docs you're looking at. So, this will end up being `timescale/timescaledb:2.0.0-rc1-pg12`

@solugebefola @tedsczelecki - am I correct on that?